Update ChangeLog (#1612)
slint-ui/slint: CHANGELOG.md

## Unreleased

### Changed
 - `mod` now works on any numeric type, not only integers.
 - Minimum rust version is now 1.60
 - The "backend-gl-*" Rust crate features for configuring the GL renderer have been
   changed and split by renderer
 - `VecModel::remove` now return the removed element
 - `slint::invoke_from_event_loop` and `slint::quit_event_loop` now return a Result

### Added

 - Added the `platform` module providing API to use slint on bare metal with a software renderer

>  - Experimental Skia renderer

Reviewer: ```suggestion
 - Added an experimental Skia renderer
```

(we use "added" or "add" for the other list items as well)

 - Added `slint::Window::scale_factor()` as getter to read the system device pixel ratio.
 - Added support for `slint::Window::set_size` and `slint::Window::size` to set and get the
   size of the window on the screen.
 - Added `slint::Window::dispatch_event` and `slint::WindowEvent` to be able to manually
   send a mouse or touch event to a window.
 - Added `animation-tick()`
 - `SharedString` implements `std::fmt::Write` and added `slint::format!`
 - `Image` can now be rotated with the `rotation-*` properties

>  - Use docking widgets and integration of slint-lsp into the Online Code Editor

Reviewer: ```suggestion
 - Use docking widgets and integration of slint-lsp into the [Online Code Editor](https://slint-ui.com/editor).
```

### Fixed
 - Fixed Ctrl+Backspace/Ctrl+Del not deleting words in text input elements.

>  - Resizing of preview window from our editor extension

Reviewer: ```suggestion
 - Resizing of live-preview window in the IDE integrations.
```

(don't need to mention that it is "ours" as ownership is not relevant; Re-use the term "live-preview" that we've used previously; same for IDE integrations)

 - Preferred size of the TabWidget in the fluent style take in account the size of the tabs (#1363)
 - Fixed cursor behavior when typing the Enter key at the end of a TextEdit (#1318)

>  - Fixed a leak on image resources with the GL backend

Reviewer: ```suggestion
 - Fixed a memory leak of images when using the GL backend.
```
